README.md

```diff
@@ -24,7 +25,11 @@ const pluginOptions = {
   applyToQueryString: true,
 };
 
-httpClient.use(oniyiHttpPluginUrlTemplate(pluginOptions));
+const plugin = oniyiHttpPluginUrlTemplate(pluginOptions);
+
+const httpClient = OniyiHttpClient
+  .create(options) // create custom http client with defined phases lists
+  .use(plugin);    // mount a plugin
```

Comment:
you can add `options` here to map the plugin's phase name to one of the existing names in the httpClient's phaseList.

see [mapping code](https://github.com/benkroeger/oniyi-http-client/blob/master/lib/helpers/mount-plugin.js#L51)

```js
httpClient.use(plugin, { requestPhaseMap: { 'format-url-template': 'initial' } });
```

Reply:
nice, forgot about that option!